src/hpp/gepetto/viewer.py

```diff
@@ -136,8 +136,12 @@ def __init__ (self, problemSolver, viewerClient = None, ghost = False, collision
             self.client.gui.addArrow("Vec_Acceleration",self.arrowRadius,self.arrowMinSize,self.colorAcceleration)
             self.client.gui.addToGroup("Vec_Acceleration",self.sceneName)
             self.client.gui.setVisibility("Vec_Acceleration","OFF")
-        self.amax = omniORB.any.from_any(self.problemSolver.client.problem.getParameter("Kinodynamic/accelerationBound"))
-        self.vmax = omniORB.any.from_any(self.problemSolver.client.problem.getParameter("Kinodynamic/velocityBound"))
+          self.amax = omniORB.any.from_any(self.problemSolver.hppcorba.problem.
```

Comment:
The indentation seem not to match the one from line 138. Is github not showing the white spaces correctly ?

Reply:
I simply followed @pFernbach instructions from https://github.com/humanoid-path-planner/hpp-gepetto-viewer/issues/37.
> Yes it work for me. But actually I introduced an indentation error in my previous pull request. The two lines that set amax and vmax should be inside the if displayArrows :.

Reply:
Yes, it's working correctly for me. Thanks and sorry about that.

Reply:
I was just surprised of the half indentation but I guess that has been checked.